src/main/java/edu/ucsb/cs156/example/controllers/UCSBDiningCommonsMenuItemController.java

```diff
@@ -62,4 +63,25 @@ public UCSBDiningCommonsMenuItem postMenuItem(
 
     return savedCommons;
   }
+
+  /**
+   * Get a single date by id
+   *
+   * @param id the id of the date
+   * @return a UCSBDate
+   */
+  @Operation(summary = "Get a single date")
```

Comment:
Summary should be updated.
```suggestion
  @Operation(summary = "Get a single menu item")
```